src/components/Loader/Loader.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Loader({ imgSrc, style = {} }) {
```

Comment:
For the naming, `Loader` reminds me something from webpack :P 
Maybe `LoadingSpinner` would be clearer?